services/motion/builtin/builtin.go

```diff
@@ -70,60 +69,32 @@ func (c *Config) Validate(path string) ([]string, error) {
 	return []string{framesystem.InternalServiceName.String()}, nil
 }
 
+type builtIn struct {
+	resource.Named
+	resource.TriviallyCloseable
+	resource.AlwaysRebuild
+	logger          golog.Logger
+	fsService       framesystem.Service
+	movementSensors map[resource.Name]movementsensor.MovementSensor
+	slamServices    map[resource.Name]slam.Service
+	components      map[resource.Name]resource.Resource
+}
+
 // NewBuiltIn returns a new move and grab service for the given robot.
 func NewBuiltIn(ctx context.Context, deps resource.Dependencies, conf resource.Config, logger golog.Logger) (motion.Service, error) {
 	ms := &builtIn{
 		Named:  conf.ResourceName().AsNamed(),
 		logger: logger,
 	}
 
-	if err := ms.Reconfigure(ctx, deps, conf); err != nil {
-		return nil, err
-	}
-	return ms, nil
-}
-
-func newFilePathLoggerConfig(filepath string) zap.Config {
```

Comment:
This diff looks really weird but effectively all this PR does is delete the reconfigure function and move the internals here, which somehow gets captured as moving the function that exists between the two of them